doc/source/conf.py

```diff
@@ -133,6 +142,8 @@
     rst_epilog += f.read()
 
 
+linkcheck_exclude_documents = ["changelog"]
```

Comment:
Why is this required?

Reply:
I noticed a 502 error for link checks in the changelog file—likely a temporary issue due to server overload https://github.com/ansys/ansys-sphinx-theme/actions/runs/11496793507/job/31999155151 , I ignored it for the moment, but I checked, and it’s no longer an issue, so I’ll revert the changes.